net/relayd/src/opnsense/Relayd.xml

```diff
@@ -0,0 +1,326 @@
+<model>
```

Comment:
should this be here under src/opnsense ?

Reply:
no, was a backup file
deleted with the third commit
ignore it please